@navikt/ds-css/baseline/baseline.css

```diff
@@ -6,7 +6,26 @@
 @import "print.css";
 @import "utility.css";
 
+@keyframes smoothscroll1 {
+  from,
+  to {
+    scroll-behavior: smooth;
+  }
+}
+
+@keyframes smoothscroll2 {
+  from,
+  to {
+    scroll-behavior: smooth;
+  }
+}
+
+html {
+  animation: smoothscroll1 1s;
+}
+
```

Comment:
Fikset slik at smoothscroll fungerer med anchorlenker, men ikke med ctrl + f sidesøk.
Dette var fiksen jeg fant på det, men klarer ikke helt se grunnen til to forskjellige keyframes her... hva har jeg misset?

Reply:
Totalt uforståelig kode, men så lenge det funker er jeg fornøyd. 

For å gjøre dette enklere å forstå ville jeg flyttet all den relevante koden til en egen fil med et smart navn, og lagt inn en kommentar med lenke til blogposten: https://schepp.dev/posts/smooth-scrolling-and-page-search/